### DISCUSSION TITLE
We now have a Discord!

### DISCUSSION THREAD

**Discussion by elementbound** [foxssake/netfox#227] [Announcements]
Some of you have mentioned that it would be nice to have a place where you can chat easily with people using netfox. Well, it is here now!

https://discord.gg/xWGh4GskG5

This server will be home to all Fox's Sake projects, including games and gamedev tools. We take your interests into account, so you'll only see stuff that you want to see :slightly_smiling_face: Hope to see you there!

**Comment by OwensC** [comment]
Link expired :(

**Reply by elementbound** [reply]
Thanks for raising, fixed! You can join at https://discord.gg/xWGh4GskG5 if you want!
